mastracode/factory-ui/src/ui/domains/factory/components/cardMorph.css

```
@@ -1,99 +1,214 @@
 
-.board-card-details[data-slot='popover-content'][data-closed] {
-  animation: board-card-unmorph 200ms cubic-bezier(0.4, 0, 1, 1) forwards;
+.board-card-actions > * {
+  position: relative;
 }
 
-.board-card-details[data-open] [data-card-morph='reveal'] {
-  animation: board-card-reveal 200ms 60ms cubic-bezier(0.22, 1, 0.36, 1) backwards;
+.board-card-actions > * + * {
+  margin-left: calc(-1 * var(--board-pill-tuck));
+  /* The button's own sm padding, then the tuck. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)